theme-components/vertical-full-page-map/script.js

```diff
@@ -32,9 +32,9 @@ ANSWERS.addComponent('VerticalFullPageMapOrchestrator', Object.assign({},
         {{#with (lookup verticalsToConfig verticalKey)}}
         {{#if isFirst}}isFirst: {{isFirst}},{{/if}}
         {{#if icon}}icon: "{{{icon}}}",{{/if}}
-        {{#if iconUrl}}iconUrl: "{{#unless (isNonRelativeUrl iconUrl)}}{{relativePath}}/{{/unless}}{{{iconUrl}}}",{{/if}}
+        {{#if iconUrl}}iconUrl: "{{{relativePathHandler url=iconUrl relativePath=@root.relativePath}}}",{{/if}}
```

Comment:
this is unrelated to this, but we can probably switch over all of the `isNonRelativeUrl` calls to use the `relativePathHandler`, right?

Reply:
yeah, this one is a little weird because the url is @key concatenated with ".html" but we have a concat helper that can do that.

Reply:
Hmm actually, we already updated these in other components that have verticalPages config. We don't want to use a concat helper here, unlike what I originally thought, because by default we want es/locations.html to have links to "people.html" not "../people.html" (if that makes sense)